### ISSUE TITLE
Get CellRanger to successfully install on docker container

### ISSUE THREAD

**Issue by cansavvy** [cansav09/scRNA-seq_workflow#23]
In working on #21, I'm working on obtaining the original tabula muris fastq files (which are not on SRA). It was recommended to us by the CZI people to download the bam files from aws and then convert them back to fastq files using cellranger's `bamtofastq`. 

Cellranger doesn't work with Mac, so I need to install `cellranger` into my docker container here. I've added to my Dockerfile what I think should work to make `cellranger` useable.  The docker container supposedly builds successfully, but thus far it is not working. Will likely need @kurtwheeler's help when he is feeling better and not busy.  

```
# Install cellranger
RUN cd /tmp/ && \
	wget -O cellranger-3.0.2.tar.gz "http://cf.10xgenomics.com/releases/cell-exp/cellranger-3.0.2.tar.gz?Expires=1550715688&Policy=eyJTdGF0ZW1lbnQiOlt7IlJlc291cmNlIjoiaHR0cDovL2NmLjEweGdlbm9taWNzLmNvbS9yZWxlYXNlcy9jZWxsLWV4cC9jZWxscmFuZ2VyLTMuMC4yLnRhci5neiIsIkNvbmRpdGlvbiI6eyJEYXRlTGVzc1RoYW4iOnsiQVdTOkVwb2NoVGltZSI6MTU1MDcxNTY4OH19fV19&Signature=QXtAQYaZ5IJdrzA1jp78ETZWOEr2wzZltEYD2J8J~-apqM4E2z1P1P1QNMKkXSGGvWO5vx8jCDJDpluNw7M1jk9KzNMtmn1blQ0fPd8w-FRxkB5CicER1Me6oFaiepiHr3TV2vhFZlIfAvAytIBAL9EvTFgqJa8IYuYTpLOvbWhZrhk4nsG8bAoAD-RqYAHQcB2yyzoklOoJaKW1ihmL1def80az~xTLm3pmswyYhNNYlvhlbyVy2ameb1r5XqfKwW0wjp28Uj2c5cTs2GVKa9iWOLdOMwo5V5s6Dxi7NgZoE3JsPr44HfkCMbFFAKLy6KqIseXP7cO38uOXbYondQ__&Key-Pair-Id=APKAI7S6A5RYOXBWRPDA" && \	
	mv cellranger-3.0.2.tar.gz /opt/ && \
	cd /opt/ && \
	tar -xzvf cellranger-3.0.2.tar.gz && \
	rm -f cellranger-3.0.2.tar.gz

# Set path
ENV PATH /opt/cellranger-3.0.2:$PATH

# Download extra plug in for bamtofastq function
RUN wget http://cf.10xgenomics.com/misc/bamtofastq && \
    chmod 700 bamtofastq
```

**Comment by cansavvy** [comment]
`Cellranger` and `bamtofastq` is working. Don't know why it wasn't working yesterday, but now it is. Saved and pushed Dockerfile. We are rocking and rolling now.

**Comment by cansavvy** [comment]
This issue was addressed by #24
